modules/checkout/tests/src/Functional/CheckoutFlowTest.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace Drupal\Tests\commerce_checkout\Functional;
+
+use Drupal\Tests\commerce\Functional\CommerceBrowserTestBase;
+use Drupal\commerce_checkout\Entity\CheckoutFlow;
+
+/**
+ * Tests the checkout flow UI.
+ *
+ * @group commerce
+ */
+class CheckoutFlowTest extends CommerceBrowserTestBase {
+
+  /**
+   * Modules to enable.
+   *
+   * @var array
+   */
+  public static $modules = [
+    'commerce_checkout',
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function getAdministratorPermissions() {
+    return array_merge([
+      'administer checkout flows',
+    ], parent::getAdministratorPermissions());
+  }
+
+  /**
+   * Tests creating a checkout flow.
+   */
+  public function testCheckoutFlowCreation() {
+    $this->drupalGet('admin/commerce/config/checkout-flows');
+    $this->getSession()->getPage()->clickLink('Add checkout flow');
+    $edit = [
+      'label' => 'Test checkout flow',
+      'id' => 'test_checkout_flow',
+      'plugin' => 'multistep_default',
+    ];
+    $this->submitForm($edit, 'Save');
+
+    // Test UI shows the correct message.
```

Comment:
I like to remove comments like these, because they generally don't say anything new, the code is already obvious. Comments are best when they explain the "why" behind the code, not repeat what the code is saying.